tsql/TSqlParser.g4

```diff
@@ -3555,155 +3584,424 @@ id
 simple_id
```

Comment:
Good job. How did you find out that these tokens can be also identifiers?

Reply:
We’ve got an rather good list of reserved words, I’m attaching it.
[reserved_words.txt](https://github.com/antlr/grammars-v4/files/1502379/reserved_words.txt)